docs/references/calfits_memo.tex

```diff
@@ -12,177 +13,155 @@
-with the code interface. Here, we describe the required and optional parameters
-of the \textit{calfits} format and interface to reading and writing, in addition
-to the structure of the underlying fits file. For examples please see the
-pyuvdata tutorial on ReadTheDocs: \url{http://pyuvdata.readthedocs.io/en/latest/tutorial.html}.
-
-\section{Basics}
-
-The \textit{calfits} format is a specification for the storage of radio
-interferometer calibration information in a fits file.  The contents of the file
-are defined via explicit mapping to the UVCal object in the pyuvdata package.
-The UVCal object (really, it's a class) is a subclass of the of the UVBase class
-with a set of \textbf{uvparameters} that defines the UVCal object. A
-\textbf{uvparameter} is a pyuvdata object that has a name, form, description,
-value, required flag, expected type, acceptable values, and tolerances
-associated with it.  \textbf{Uvparameters} are accessed as attributes to the
-UVBase class (and its subclasses, like UVCal).
-
-\section{Parameters}
-In order to conform to the \textit{calfits} file format there are a number of
-required parameters that need to be set as attributes of the UVCal class. Below
-is a list of the required parameters and their descriptions.
-
+This memo introduces a new FITS-based file format for storing calibration solutions to use with pyuvdata\footnote{\url{https://github.com/RadioAstronomySoftwareGroup/pyuvdata}}, a python package which
+provides a software interface for interferometry.
+% We are defining a file format with the code interface. 
+We describe the required and optional parameters of a UVCal FITS---hereafter \textit{calfits}---file and pyuvdata's interface for reading and writing these files. 
+For usage examples please refer to the pyuvdata tutorial: \url{http://pyuvdata.readthedocs.io/en/latest/tutorial.html}.
+
+\section{Overview}
+\textit{Calfits} is an adaptation of the FITS file format to enable the storage of calibration information for radio interferometric arrays.
+Because it builds on the existing FITS file format, all \emph{calfits} files should be properly formatted according to the FITS standard, which is widely available\footnote{\url{https://fits.gsfc.nasa.gov/fits_documentation.html}}.
+
+
+Any valid \textit{calfits} file corresponds directly to a UVCal object within pyuvdata.
+As such, every new HDU keyword introduced here has a one-to-one correspondence with UVCal object parameters.
+In this memo, each new keyword is followed by its corresponding UVCal parameter, in parentheses. 
+For more information about the UVCal class and its parameters, please refer to pyuvdata's documentation: \url{http://pyuvdata.readthedocs.io/en/latest/uvcal.html}.
+
+A UVCal object stores calibration solutions as either a ``gain''-type solution, or as a ``delay''-type solution.
+These types represent two distinct but mathematically equivalent conventions, both widely used in radio astronomy.
```

Comment:
They aren't really mathematically equivalent -- delays have much less information. Delays can be converted to gains but gains cannot be converted to delays because they do not have to behave as if there is just a delay per antenna. I'd just take out the "mathematically equivalent" clause here.